packages/spright-components/src/chat/specs/README.md

```diff
@@ -0,0 +1,252 @@
+    </spright-chat-message>
+    <spright-chat-message status="outgoing">
+        I need to analyze my data to find anomalies.
+    </spright-chat-message>
+    <spright-chat-message status="system">
+        <nimble-spinner></nimble-spinner>
+    </spright-chat-message>
+</spright-chat-conversation>
+```
+
+#### Rich text message example
+
+```html
+<spright-chat-conversation>
+    <spright-chat-message status="incoming">
+        <nimble-rich-text-viewer id="welcome"></nimble-rich-text-viewer>
+    </spright-chat-message>
+</spright-chat-conversation>
+```
+
+```js
+const richText = document.querySelector('#welcome');
+richText.markdown = 'Welcome **Homer**, how can I help?';
+```
+
+#### Prompt buttons message example
+
+```html
+<spright-chat-message status="system">
+    <nimble-button appearance="block">Help with my taxes</nimble-button>
+    <nimble-button appearance="block">Provide me some life advice</nimble-button>
+</spright-chat-message
+```
+
+### API
+
+#### Message
+
+- _Component Name_ `spright-chat-message`
+- _Props/Attrs_
+    - `status = "incoming" | "outgoing" | "system"`
+- _Methods_
+- _Events_
+- _CSS Classes and CSS Custom Properties that affect the component_
+- _How native CSS Properties (height, width, etc.) affect the component_
+    - A message will grow its width to fit its content, up to a maximum width.
+    - A message will grow its height to fit its content, with no maximum height.
+    - Clients could override this behavior but we don't anticipate use cases for doing so when the message is used within a conversation
+- _Slots_
+
+    - arbitrary content can be added to the default slot to be displayed within the message
+
+#### Conversation
+
+- _Component Name_ `spright-chat-conversation`
+- _Props/Attrs_
+- _Methods_
+- _Events_
+- _CSS Classes and CSS Custom Properties that affect the component_
+- _How native CSS Properties (height, width, etc.) affect the component_
+    - Clients can size the conversation using normal CSS rules.
+    - The conversation will show a scrollbar if content overflows vertically.
+    - The conversation will have a minimum width that clients are discouraged from overriding.
+- _Slots_
+    - chat messages are added to the default slot. The DOM order of the messages controls their screen order within the conversation (earlier DOM order => earlier message => top of the conversation)
+
+### Anatomy
+
+#### Message
+
+A message is simply a `div` which will styled with background / border / rounded corners and the default slot for the message contents.
+
+```html
+<template>
+    <div>
+        <slot></slot>
+    </div>
+</template>
+```
+
+#### Conversation
+
+Other than setting a background, a conversation has no appearance of its own and simply contains the default slot for messages.
+
+```html
+<template>
+    <slot></slot>
+</template>
+```
+
+### Native form integration
+
+Native form integration is not needed for these components.
+
+### Angular integration
```

Comment:
My hunch is that the framework integration will be trivial since the API of these components is trivial but I didn't want to make that claim without digging into it. If any reviewers want to weigh in to confirm/deny that hunch, I'm happy to update these sections.

Reply:
I don't think there are any complexities to be concerned about like router integration, form integration, or non-primitive property binding. Following the standard stateless directive pattern should be expected.